### PR TITLE
Implemented resend activation link and delete end points

### PR DESCRIPTION
**Pull request by tgreyuk**

### CODE REVIEW

**Comment by tgreyuk** [comment]
[(6) User Accounts - Pending User - Resend Activation Link](https://trello.com/c/HlxXnamI/141-6-user-accounts-pending-user-resend-activation-link)

File: src/app/core/services/user.service.ts [hunk 1]
```diff
@@ -48,19 +48,17 @@ export class UserService {
...
 
-  /*
-   * TODO - Implement API calls once BE ready
-   */
-  public resendActivationLink(useruid: string, activationuid: string) {
-    return of({});
+  public resendActivationLink(useruid: string) {
+    const token = localStorage.getItem('auth-token');
+    const headers = new HttpHeaders({ Authorization: token });
+    return this.http.post(`api/user/${useruid}/resend-activation`, null, {
```
Comment (ChazUK) [inline src/app/core/services/user.service.ts]: This needs to be /api/ and then you don’t need to worry about adding the Bearer Token as it’ll be handled by the Auth HTTP Interceptor

File: src/app/core/services/user.service.ts [hunk 2]
```diff
@@ -48,19 +48,17 @@ export class UserService {
...
+      headers,
+      responseType: 'text' as 'json',
```
Comment (ChazUK) [inline src/app/core/services/user.service.ts]: Same with this type

File: src/app/core/services/user.service.ts (hunk 1 above)
Comment (tgreyuk) [reply src/app/core/services/user.service.ts]: Well that's what I thought but it wasn't authorizing without it. I will look into it further...

File: src/app/core/services/user.service.ts (hunk 2 above)
Comment (tgreyuk) [reply src/app/core/services/user.service.ts]: WIthout the type - it fails to return a response - again must be an issue with the interceptor and POST or something,

Comment (tgreyuk) [reply src/app/core/services/user.service.ts]: Ok the issue was the regx in the incerptor was blocking the interceptor as i didnt have '/' in front of api. Have removed the token.

Comment (tgreyuk) [reply src/app/core/services/user.service.ts]: Ok the issue was the regx in the incerptor was blocking the interceptor as i didnt have '/' in front of api. Have removed the token. But i still fails without the response type here.

Comment (tgreyuk) [reply src/app/core/services/user.service.ts]: It is because the reponse is empty. If you don't set response type as text it tries to parse it as json.

**Pull request merged**